cmd/buildah/common.go

```diff
@@ -239,9 +261,9 @@ func getFormat(c *cli.Context) (string, error) {
 	}
 }
 
-func sortFlags(flags []cli.Flag) []cli.Flag {
-	sort.Slice(flags, func(i, j int) bool {
-		return strings.Compare(flags[i].GetName(), flags[j].GetName()) < 0
-	})
-	return flags
-}
+//func sortFlags(flags []cli.Flag) []cli.Flag {
```

Comment:
Is this still needed or are flags automatically sorted?

Reply:
automatically

Reply:
Then remove the code.

Reply:
i did